tests/load/build_path_lists.sh

```diff
@@ -0,0 +1,47 @@
+sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COLLISION_TABULATION\&id=\1%2F\2\&format=WEB/' features/centreline.txt > paths/getReportWeb_COLLISION_TABULATION.txt
+grep -E "^[12467]" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_24H\&id=\1%2F\2\&format=PDF/' > paths/getReport_COUNT_SUMMARY_24H.txt
+grep -E "^[12467]" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_24H\&id=\1%2F\2\&format=WEB/' > paths/getReportWeb_COUNT_SUMMARY_24H.txt
+grep -E "^[12467]" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_24H_DETAILED\&id=\1%2F\2\&format=PDF/' > paths/getReport_COUNT_SUMMARY_24H_DETAILED.txt
+grep -E "^[12467]" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_24H_DETAILED\&id=\1%2F\2\&format=WEB/' > paths/getReportWeb_COUNT_SUMMARY_24H_DETAILED.txt
+grep -E "^[12467]" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_24H_GRAPHICAL\&id=\1%2F\2\&format=PDF/' > paths/getReport_COUNT_SUMMARY_24H_GRAPHICAL.txt
+grep -E "^[12467]" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_24H_GRAPHICAL\&id=\1%2F\2\&format=WEB/' > paths/getReportWeb_COUNT_SUMMARY_24H_GRAPHICAL.txt
+grep -E "^5" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_TURNING_MOVEMENT\&id=\1%2F\2\&format=PDF/' > paths/getReport_COUNT_SUMMARY_TURNING_MOVEMENT.txt
+grep -E "^5" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_TURNING_MOVEMENT\&id=\1%2F\2\&format=WEB/' > paths/getReportWeb_COUNT_SUMMARY_TURNING_MOVEMENT.txt
+grep -E "^5" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_TURNING_MOVEMENT_DETAILED\&id=\1%2F\2\&format=PDF/' > paths/getReport_COUNT_SUMMARY_TURNING_MOVEMENT_DETAILED.txt
+grep -E "^5" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=COUNT_SUMMARY_TURNING_MOVEMENT_DETAILED\&id=\1%2F\2\&format=WEB/' > paths/getReportWeb_COUNT_SUMMARY_TURNING_MOVEMENT_DETAILED.txt
+grep -E "^5" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=INTERSECTION_SUMMARY\&id=\1%2F\2\&format=PDF/' > paths/getReport_INTERSECTION_SUMMARY.txt
+grep -E "^5" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=INTERSECTION_SUMMARY\&id=\1%2F\2\&format=WEB/' > paths/getReportWeb_INTERSECTION_SUMMARY.txt
+grep -E "^4" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=SPEED_PERCENTILE\&id=\1%2F\2\&format=PDF/' > paths/getReport_SPEED_PERCENTILE.txt
+grep -E "^4" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=SPEED_PERCENTILE\&id=\1%2F\2\&format=WEB/' > paths/getReportWeb_SPEED_PERCENTILE.txt
+grep -E "^5" features/counts.txt | sed -r 's/^(.*)\t(.*)$/\/reporter\/reports?type=WARRANT_TRAFFIC_SIGNAL_CONTROL\&id=\1%2F\2\&format=PDF/' > paths/getReport_WARRANT_TRAFFIC_SIGNAL_CONTROL.txt
```

Comment:
these endpoints probably need additional parameters - the `status` error rates are very high.  (this also means possibly redoing this particular test.)